tests/functional/spec/relationships.spec.js

```diff
@@ -107,109 +114,85 @@ describe('Relationships', function() {
       beforeEach('add three people and complete their relationships', function() {
         addThreePeopleAndCompleteRelationships();
       });
 
-      it('Then I delete one of the original household members I will not be asked for the original members relationships again', function() {
-        browser.url('/questionnaire/list-collector');
-        $(ListCollectorPage.listRemoveLink(3)).click();
+      it('Then I delete one of the original household members I will not be asked for the original members relationships again', function(){
+        $(SectionSummaryPage.listCollectorPeopleRowRemove(1)).click();
         $(ListCollectorRemovePage.yes()).click();
         $(ListCollectorRemovePage.submit()).click();
-        $(ListCollectorPage.no()).click();
-        $(ListCollectorPage.submit()).click();
-        expect(browser.getUrl()).to.contain('/questionnaire/confirmation');
+        expect(browser.getUrl()).to.contain('/sections/section/');
       });
 
-      it('Then I will be asked for the original household members relationships to all new members', function() {
-        browser.url('/questionnaire/list-collector');
-        $(ListCollectorPage.yes()).click();
-        $(ListCollectorPage.submit()).click();
+      it('Then I add another household member I will be asked for about all relationships', function(){
+        $(SectionSummaryPage.listCollectorPeopleRowAdd()).click();
         $(ListCollectorAddPage.firstName()).setValue('Tom');
         $(ListCollectorAddPage.lastName()).setValue('Bowden');
         $(ListCollectorAddPage.submit()).click();
-        $(ListCollectorPage.no()).click();
-        $(ListCollectorPage.submit()).click();
         expect($(RelationshipsPage.husbandOrWife()).isSelected()).to.be.true;
         $(RelationshipsPage.submit()).click();
         expect($(RelationshipsPage.legallyRegisteredCivilPartner()).isSelected()).to.be.true;
         $(RelationshipsPage.submit()).click();
         expect($(RelationshipsPage.playback()).getText()).to.contain('Tom Bowden is Marcus Twin’s …');
-
         $(RelationshipsPage.sonOrDaughter()).click();
         $(RelationshipsPage.submit()).click();
         expect($(RelationshipsPage.husbandOrWife()).isSelected()).to.be.true;
         $(RelationshipsPage.submit()).click();
         expect($(RelationshipsPage.playback()).getText()).to.contain('Tom Bowden is Samuel Clemens’ …');
-
         $(RelationshipsPage.sonOrDaughter()).click();
         $(RelationshipsPage.submit()).click();
         browser.pause(20000);
         expect($(RelationshipsPage.playback()).getText()).to.contain('Tom Bowden is Olivia Clemens’ …');
       });
-
-      it('When I add a household member after answering the relationships question but remove them again before exiting the list-collector block, Then I will not be asked for the original household members relationships to the new member', function() {
```

Comment:
Due to the way we now explicitly go back to the summary only if its the last block or the return_to_summary flag is set. In this instance, as the respondent is going back and forward in the section it won't go back to the summary page now and this test become redundant